### PR TITLE
Support platform specific reftests.

### PR DESCRIPTION
**Pull request by glennw**
Enable the synthetics italics reftest on Linux.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/1752)

### CODE REVIEW

File: wrench/src/reftest.rs [hunk 1]
```diff
@@ -19,6 +19,15 @@ use webrender::api::*;
 use wrench::{Wrench, WrenchThing};
 use yaml_frame_reader::YamlFrameReader;
 
+#[cfg(target_os = "windows")]
+const PLATFORM: &str = "win";
+#[cfg(target_os = "linux")]
```
Comment (kvark) [inline wrench/src/reftest.rs]: Do we alienate non-linux UNIX systems into `other` category?
Perhaps, we can use `cfg(windows)`, `cfg(unix)`, and `cfg(target_os="macos")` instead, see [target flags](https://doc.rust-lang.org/reference/attributes.html#conditional-compilation) for more info

File: wrench/src/reftest.rs [hunk 2]
```diff
@@ -19,6 +19,15 @@ use webrender::api::*;
...
+const PLATFORM: &str = "linux";
+#[cfg(target_os = "macos")]
+const PLATFORM: &str = "mac";
+#[cfg(not(any(target_os = "linux", target_os = "macos", target_os = "windows")))]
```
Comment (kvark) [inline wrench/src/reftest.rs]: sometimes I wish there was `if-else` in `cfg` things

File: wrench/src/reftest.rs (hunk 1 above)
Comment (glennw) [reply wrench/src/reftest.rs]: Yea - you're right. I'll update it to be unix.

**Pull request merged**